File: CS/CustomProviderSample/frmDEsigner.cs
using System.Windows.Forms;
using DevExpress.XtraReports.Wizards;
// ...

namespace CustomProviderSample {
    public partial class frmDesigner : Form {
        public frmDesigner() {
            InitializeComponent();
            reportDesignerMDIController.AddService(typeof(IWizardCustomizationService), new MyWizardCustomizationService());
            Load += FrmDesigner_Load;
        }

        void FrmDesigner_Load(object sender, System.EventArgs e) {
            reportDesignerMDIController.CreateNewReportWizard();
        }
    }
}

File: VB/CustomProviderSample/frmDEsigner.vb
Imports System.Windows.Forms
Imports DevExpress.XtraReports.Wizards3
' ...

Namespace CustomProviderSample
    Partial Public Class frmDesigner
        Inherits Form

        Public Sub New()
            InitializeComponent()
            reportDesignerMDIController.AddService(GetType(IWizardCustomizationService), New MyWizardCustomizationService())
        End Sub
    End Class
End Namespace
